ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml

```diff
@@ -94,6 +106,17 @@ tests:
       TEMPEST_CONF_OVERRIDES: share.run_share_group_tests False share.capability_storage_protocol
         CEPHFS
     workflow: openstack-k8s-operators-test-build-deploy-tempest
+- as: manila-operator-build-deploy-kuttl
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  optional: true
+  steps:
+    workflow: openstack-k8s-operators-test-build-deploy-kuttl
```

Comment:
Sorry forgot to mention that we need:
```
    env:
      CEPH_HOSTNETWORK: "false"
```
to run on 4.12, since it is a 3-node cluster
so it should end up like:
```
  steps:
    env:
      CEPH_HOSTNETWORK: "false"
    workflow: openstack-k8s-operators-test-build-deploy-kuttl
```

Reply:
ah; nice